newm-server/src/main/kotlin/io/newm/server/database/migration/V32__ConfigUpdates.kt

```diff
@@ -0,0 +1,20 @@
+package io.newm.server.database.migration
+
+import io.newm.server.config.repo.ConfigRepository.Companion.CONFIG_KEY_EVEARA_STATUS_CHECK_REFIRE
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+@Suppress("unused")
+class V32__ConfigUpdates : BaseJavaMigration() {
+    override fun migrate(context: Context?) {
+        transaction {
+            execInBatch(
+                listOf(
```

Comment:
Since there is only one line, instead of `execInBatch(listOf(...))` you can just do `exec(...)`